### ISSUE TITLE
Exile Window from Jeska's Will Stays Around After A Rollback

### ISSUE THREAD

**Issue by Alex-Vasile** [magefree/mage#8686]
Steps:
1. I cast [[Jeska's Will]] which was exiled through the cascade ability on [[Bituminous Blast]].
2. I get an exile window with 3 cards in it. I play one of the cards.
3. I roll back

The exile window remains there, and the cards still say they're playable (they can't be played if you try).

**Comment by jeffwadsworth** [comment]
Yeah, some exile effects aren't cleared via rollback.  I will check it out because I was recently working on a card that worked correctly in this regard.

**Comment by jeffwadsworth** [comment]
Ok, check the card [[Glimpse the Cosmos]] and verify for me that it works correctly when rolled back.  It does for me, but I have seen some weird stuff on other clients in the past.

**Comment by jeffwadsworth** [comment]
https://github.com/magefree/mage/issues/6856

**Comment by jeffwadsworth** [comment]
I would assume that the issue is related to the process clearing the board after a rollback.  The exile zones with exileIds assigned are just being missed in the clearing routine.  But, finding this method has proven to be a needle in a haystack.  The board state has the exile zone information correctly set after a rollback.  The exile window is just a garbage artifact.
